secretstores/aws/secretmanager/secretmanager.go

```diff
@@ -86,6 +88,11 @@ func (s *smSecretStore) GetSecret(req secretstores.GetSecretRequest) (secretstor
 	return resp, nil
 }
 
+// GetSecrets retrieves all secrets in the store and returns a map of decrypted string/string values
+func (s *smSecretStore) GetSecrets(req secretstores.GetSecretsRequest) (secretstores.GetSecretsResponse, error) {
+	return secretstores.GetSecretsResponse{Data: nil}, status.Errorf(codes.Unimplemented, "method SaveState not implemented")
```

Comment:
Do you have a reason why you use grpc status for error?

Reply:
Obviously a copy & paste error. I'll push a fix.